profiler/test/Datadog.Profiler.IntegrationTests/Helpers/EnvironmentHelper.cs

```diff
@@ -268,6 +268,26 @@ private static string GetMonitoringHome()
             return s;
         }
 
+        private static string GetProfilerHomeDirectory()
+            => IsRunningInCi()
+                    ? Path.Combine(GetMonitoringHome(), GetArchitectureSubfolder())
+                    : GetDeployDir();
```

Comment:
🙁 I was trying to get rid of the entire concept of "profiler home", as it just adds confusion IMO 🙁

Reply:
I know, but it prevents the tests from running in VS.
We want the freshly compiled binary in VS used in the integration tests in VS.